pkgs/stdenv/adapters.nix

```diff
@@ -238,4 +238,23 @@ rec {
         allowSubstitutes = false;
       });
     });
+
+
+  /* Modify a stdenv so that it builds binaries with the specified list of
+     compilerFlags appended and passed to the compiler.
+
+     This example would recompile every derivation on the system with
+     -funroll-loops and -O3 passed to each gcc invocation.
+
+     Example:
+       nixpkgs.overlays = [
+         (self: super: { stdenv = super.withCFlags { stdenv = super.stdenv; compilerFlags = [ "-funroll-loops" "-O3" ]; }; })
```

Comment:
Docs are outdated.

Reply:
Forgot to do it in the code, but did it in the issue hah. Thanks.